examples/graphene_apps/python_workloads/Makefile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Intel Corporation
```

Comment:
This is Amkefile for Avalon python worker. It should be kept in $TCF_HOME/tc/graphene/python_worker

Reply:
This Makefile is to build python workload module. makefile to build python worker is present in $TCF_HOME/tc/graphene/python_worker